File: src/com/first/first/java/Person.java
package com.first.first.java;

public class Person {
    private String name;
    private boolean isMarried;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public boolean isMarried() {
        return isMarried;
    }

    public void setMarried(boolean married) {
        isMarried = married;
    }

    @Override
    public String toString() {
        return "Person{" +
                "name='" + name + '\'' +
                ", isMarried=" + isMarried +
                '}';
    }
}

File: src/com/first/first/Person.kt
package com.first.first

data class Person(val name: String,
                  val age: Int? = null) // Допускаеться значение null (из за ? )

fun main(args: Array<String>) {
    val persons = listOf(Person("Alice"),
            Person("Bob",29),
            Person("Mark",30),
            Person("Dave",25))
    val oldest = persons.maxBy { it.age ?:0 } // WTF?
    // ?: - Элвис
    // если age = null то он заменяет его на 0
    print("The oldest is : $oldest");
}